.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "rainbow-dragon"
```

Comment:
I've recycled an unused old project of mine since I've hit the limit of 20 free firebase projects. The downside of this is we have the word `rainbow-dragon` in our codebase here. There upside is we can use the url `rainbow-dragon.web.app` for free for something if we ever decide that's useful 😄 .

🐉 🌈